src/Raven.Studio/wwwroot/Content/css/bootstrap/variables.less

```diff
@@ -560,7 +560,7 @@
 //##
 
 //** Tooltip max width
-@tooltip-max-width:           200px;
+@tooltip-max-width:           215px;
```

Comment:
can we scope this to dashboard only?

Reply:
css selector:
#dashboard-container .tooltip

Reply:
done